config/packages/framework.yaml

```diff
@@ -19,8 +19,9 @@ framework:
         cookie_httponly: true
         cookie_secure: true
         cookie_samesite: none
-        cookie_lifetime: '%session_max_absolute_lifetime%'
-        gc_maxlifetime: '%session_max_relative_lifetime%'
+        cookie_lifetime: 0
+        gc_maxlifetime: 0
```

Comment:
Do you  intentionally configure the cookie_lifetime and gc_maxlifetime? Do we need them configured this way?

In the original situation we relied on the default values for these settings (as in, we did not configure them)

Please double check this, as my memory does not always serve me right..